tools/src/main/scala/scala/scalanative/linker/Classpath.scala

```diff
@@ -19,22 +19,20 @@ sealed trait Path {
   def globals: Set[Global]
 }
 
-object Path {
+object Classpath {
```

Comment:
How about writing `ClassPath` instead? Then it would conform to the spelling of `scala.tools.nsc.util.ClassPath`.